07/reshape-a-matrix.checkpoint.md

```diff
@@ -231,6 +283,35 @@ Steps to solve a subproblem
 
 Write the code to solve the problem here.  You can use the tests to guide your solution.
```

Comment:
I'm suggesting this because:

1. "Write the code to solve the problem" felt a lil awkward to me
2. We discussed how students will need to change their assumptions. It'll be good to acknowledge to them that they should do that, and it doesn't invalidate their previous thoughts.

```suggestion
Implement `reshape_matrix`.

Use these tests to guide your solution. These tests may make different assumptions about the problem than you did! Do not alter your assumptions above, and take these assumptions for this challenge.
```